cpp/include/cudf/detail/aggregation/aggregation.hpp

```diff
@@ -411,6 +411,15 @@ struct target_type_impl<
   using type = int64_t;
 };
 
+// Summing fixed_point numbers, use the same type accumulator
```

Comment:
@harrism @jrhemstad Question: integer types promote to 64 bit for some reason (I assume in an attempt to avoid overflow in some cases). Should `fixed_point` follow the same behaviour? 
```
// Summing/Multiplying integers of any type, always use int64_t accumulator
```

Reply:
I think it would be most consistent to keep the behavior of using the largest available type. It's not really an arbitrary decision. It's a compromise between the fact that we don't want to require the caller to have to unnecessarily cast while we also don't want to have to enable the user to specify an output type that avoids overflow.

Reply:
Sounds good to me :+1: I didn't feel strongly, glad I double checked. Changes pushed.